apps/kampus/features/url/generate-url.ts

```diff
@@ -0,0 +1,46 @@
+import { env } from "~/env";
+
+export function getCommentURL({
+  baseUrl,
+  postID,
+  commentID,
+}: {
+  baseUrl: string;
+  postID: string;
+  commentID: string;
+}) {
+  const postURL = getPostURL({ postID, baseUrl });
+  const commentUrl = `${postURL}#c_${commentID}`;
+  return commentUrl;
+}
+
+export function getPostURL({ postID, baseUrl }: { postID: string; baseUrl: string }) {
```

Comment:
```suggestion
export function getPostURL({ postID }: { postID: string }) {
```

Reply:
we don't want anyone to accidentally create wrong urls in the application, so let's remove the opportunity to change it from outside.